src/modules/CC1101/CC1101.cpp

```diff
@@ -489,6 +489,24 @@ int16_t CC1101::setRxBandwidth(float rxBw) {
   return(RADIOLIB_ERR_INVALID_RX_BANDWIDTH);
 }
 
+int16_t CC1101::autoSetRxBandwidth() {
+    // Uncertainty ~ +/- 40ppm for a cheap CC1101
+    // Uncertainty * 2 for both transmitter and receiver
+    float uncertainty = (this->frequency) * 40 * 2);
+    uncertainty = (uncertainty/1000); //Since bitrate is in kBit
+    float minbw = (this->bitRate) + uncertainty);
```

Comment:
Looks like there's an extra parenthesis here which prevents this from being compiled - see the most recent CI run:  https://github.com/jgromes/RadioLib/actions/runs/8458982857/job/23174368214#step:6:1235

Reply:
Fixed